README.md

```diff
@@ -154,6 +155,16 @@ boris log 72 | grep INTERESTING
 boris log 72 | less
 ```
 
+#### ignore <project> <build>
+
+Ignore any failures for a specific build.
```

Comment:
"current failures" or "outstanding failures" probably. Any new failures resets the status.

Reply:
thanks, fixed.